src/backend/tests/work-packages-create.test.ts

```diff
@@ -13,14 +13,20 @@ describe('work package create', () => {
     };
 
     describe('validate inputs', () => {
+      const mockWbsNum = {
```

Comment:
I'm pretty sure there's a few of these in the test data you can use

Reply:
https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/src/test-support/test-data/wbs-numbers.stub.ts